src/pubsub/ua_pubsub_ns0.c

```diff
@@ -852,23 +859,27 @@ addWriterGroupRepresentation(UA_Server *server, UA_WriterGroup *writerGroup){
     UA_StatusCode retVal = UA_STATUSCODE_GOOD;
     if(writerGroup->config.name.length > 512)
         return UA_STATUSCODE_BADOUTOFMEMORY;
-    UA_STACKARRAY(char, wgName, sizeof(char) * writerGroup->config.name.length + 1);
+    char wgName[513];
     memcpy(wgName, writerGroup->config.name.data, writerGroup->config.name.length);
     wgName[writerGroup->config.name.length] = '\0';
-    //This code block must use a lock
-    UA_NODESTORE_REMOVE(server, &writerGroup->identifier);
-    retVal |= addPubSubObjectNode(server, wgName, writerGroup->identifier.identifier.numeric,
-                                  writerGroup->linkedConnection->identifier.identifier.numeric,
-                                  UA_NS0ID_HASCOMPONENT, UA_NS0ID_WRITERGROUPTYPE);
-    //End lock zone
+
+    UA_ObjectAttributes object_attr = UA_ObjectAttributes_default;
+    object_attr.displayName = UA_LOCALIZEDTEXT("", wgName);
```

Comment:
Why is the first part of the localizedtext not needed?

Reply:
For consistency.
If we want to have "en-US", we should have it everywhere.